onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1036,8 +1036,19 @@ Applies to session load, initialization, etc. Default is 0.)pbdoc")
                                 onnxruntime::FreeDimensionOverrideType::Name,
                                 dim_value}); },
           "Rpbdoc(Specify values of named dimensions within model inputs.)pbdoc")
-      .def_readwrite("use_prepacking", &SessionOptions::use_prepacking,
-                     R"pbdoc(Whether to use prepacking. Default is true.)pbdoc");
+      .def(
+          "add_session_config_entry",
+          [](SessionOptions* options, const char* config_key, const char* config_value) -> void {
+            if (!AddSessionConfigEntryImpl(*options, config_key, config_value))
+              throw std::runtime_error("Error calling AddSessionConfigEntryImpl");
+          },
+          "Rpbdoc(Set a single session configuration entry as a pair of strings.)pbdoc")
+      .def(
+          "get_session_config_entry",
+          [](SessionOptions* options, const char* config_key) -> std::string {
+            return options->session_configurations.at(config_key);
+          },
```

Comment:
Does this provide a useful error message if the key isn't present, or should we explicitly check and handle that ourselves?